packages/docusaurus-theme-live-codeblock/src/theme/Playground/index.tsx

```diff
@@ -115,7 +116,13 @@ export default function Playground({
   } = themeConfig as ThemeConfig;
   const prismTheme = usePrismTheme();
 
-  const noInline = props.metastring?.includes('noInline') ?? false;
+  const meta = parseCodeBlockMeta({
+    metastring: props.metastring,
+    language: undefined,
+    magicComments: [],
+  });
+
+  const noInline = meta.options.noinline === true;
```

Comment:
```suggestion
  const noInline = meta.options.noInline === true;
```

Is there a reason to not respect the case?

Reply:
Initially I wanted to make things case insensitive so that the user-errors on using options is reduced. e.g. some having codeblocks with `noInline` others `noinline`. or `ShowLineNumbers` vs. `showLineNumbers`. 

I think we have 3 options here: 

1. Allow case insensitive keys on the end-user side and ensure in the code we use lowercase keys. (pro: easy on end user side, con: sensitive on component side)
2. Be case sensitive. (pro: easy on component side: con: risky for end user side)
3. Hide the values behind a case insensitive `Map<>` like abstraction. (pro: easy to use on component and end-user side, con: higher complexity in code, not easy to specify in TSX/MDX)
 
```
type CodeMetaOptions = {
    get(key:string): CodeMetaOptionValue | undefined;
    set(key:string, value: CodeMetaOptionValue | undefined);
    delete(key:string): void;
};
```

For the sake of keeping this change simple I'll change things to (2).

Reply:
We're case sensitive again in the new implementation.